docs/versioned_docs/version-v0.38/configure-deploy.md

```diff
@@ -98,14 +98,14 @@ We will be using [httpbin.org](https://httpbin.org) as the upstream server. The
 service echoes incoming HTTP Requests and is perfect for seeing how ORY
 Oathkeeper works. Let's define three rules:
 
-1. An access rule that allowing anonymous access to
-   `https://httpbin.org/anything/cookie` and using the `cookie` mutator.
+1. An access rule allowing anonymous access to
+   `https://httpbin.org/anything/header` using the `cookie` mutator.
```

Comment:
Actually looks like we don't use the cookie mutator at all in the example below?

Reply:
To me it looked like it was just a anonymous request with some header. 
I didnt really get the cookie part, but if we dont use it at all, that makes much more sense!
```suggestion
   `https://httpbin.org/anything/header` using any header.
```

Reply:
Yes that looks correct!

Reply:
this would be correct:

```suggestion
   `https://httpbin.org/anything/header` using the `header` mutator.
```